docs/integrations/spark/spark_column_lineage.md

```diff
@@ -0,0 +1,109 @@
+---
+sidebar_position: 1
+---
+
+# Column Level Lineage
```

Comment:
Maybe this should be two docs: one about all-around column level lineage and how OpenLineage understands that regardless of integration, and second, more technical, which focuses on Spark implementation

Reply:
Yeah, had similar idea but thought it's better to have all in single place up until column level lineage gets integrated elsewhere.